### ISSUE TITLE
Unable to parse file

### ISSUE THREAD

**Issue by artrealla** [kinkofer/FightClub5eXML#177]
Can’t import any collection files to 5e, only source files. Keep getting the error “Unable to parse file.” I’m not editing the xml at all, just doing a direct download.

**Comment by zcdziura** [comment]
Can you describe a little bit more about what you're doing? Are you just downloading the source files from this repository and trying to import them directly? If so, that won't work. The README file at the root of this project details what you have to do in order to be able to import data into Fight Club 5e

**Comment by artrealla** [comment]
No, I’m attempting to use one of the collection files, but I just get the error “Unable to parse file” every time I try to use it. The pathing for the sources in the collection file are correct, and I’ve validated the xml, so I’m not sure why it’s not working. I have an iphone, and I downloaded the master straight to dropbox & moved it into my files, and am just attempting to use the core & supplements collection.
I want to note that the source files will import just fine on their own, and I know that they’re not in the right format to work as a complete compendium, but it proves I can get a successful import.

**Comment by zcdziura** [comment]
Oh, I see the issue here.

Rather than re-typing this all out again, another user had a similar problem to what you're facing. I've linked the discussion below for you to look over. I think it should address the issue you're having and provide some insight into this repository as a whole and what it provides.

https://github.com/kinkofer/FightClub5eXML/discussions/174
